plugins/storage/pyparts/ceph_daemon_checks.py

```diff
@@ -33,12 +33,30 @@ def check_require_osd_release(self):
             return
 
         for rname in cluster.daemon_release_names('osd'):
-            if expected_rname > rname:
+            if expected_rname != rname:
                 msg = ("require_osd_release is {} but one or more osds is on "
                        "release {} - needs fixing".format(expected_rname,
                                                           rname))
                 issue_utils.add_issue(issue_types.CephOSDError(msg))
 
+    def check_osd_versions(self):
```

Comment:
can you please add a docstring to explain what this is doing/checking (for the uninitiated)

Reply:
ack